Jenkinsfile

```diff
@@ -128,6 +122,21 @@ ansiColor('xterm') {
           }
 
           if (currentBuild.result == 'SUCCESS'){
+            stage('Bump version'){
+              sh '''#!/bin/bash -ex
+              source ~/.nvm/nvm.sh
+              nvm use v6
+              npm version patch
+              grep "version" package.json | head -1 | awk -F: '{ print $2 }' | sed 's/[",]//g' > .version
```

Comment:
That's actually magic for Jenkins to get the version number. The only thing we're using to bump the version is 'npm version patch'